pkg/operator/ceph/test/containers.go

```diff
@@ -84,7 +89,10 @@ func (ct *ContainersTester) AssertEnvVarsContainCephRequirements() {
 		if !isCephCommand(c.Command) {
 			continue // don't consider containers that aren't Ceph commands
 		}
-		assert.Subset(ct.t, varNames(&c), requiredEnvVars)
+		if !assert.Subset(ct.t, varNames(&c), requiredEnvVars) {
```

Comment:
This looks like it gets the tests passing, but seems awkward since it wouldn't actually require the env vars on the mgr pod spec. @BlaineEXE Shall we go with this or do you have another suggestion?

Reply:
This seems very wrong to me. The initial assert (removed here) would read in english as "assert that the set `requiredEnvVars` is a subset of `varNames`". I don't see how the inverse of this is good at all. Asserting that it *isn't* a subset means that the var names are required *not* to have all the required env vars.